builder.go

```diff
@@ -34,6 +34,7 @@ git clone --depth 1 {{.CloneURL}} repo
 cd repo
 git fetch origin {{.Ref}}
 git checkout -qf {{.Head}}
+git submodule update --init
```

Comment:
Yes it can be run on the root dir, seems like it sync every submodule inside code base.